libevm/lib/service_evm.go

```diff
@@ -37,7 +37,7 @@ type EvmParams struct {
 	To            *common.Address  `json:"to"`
 	Value         *hexutil.Big     `json:"value"`
 	Input         []byte           `json:"input"`
-	Gas           hexutil.Uint64   `json:"gas"`
+	Gas           hexutil.Uint64   `json:"availableGas"`
```

Comment:
I'm sorry, now it is working, but really confusing. Please rename the variable name too, not just the json property.